app.json

```diff
@@ -10,7 +10,9 @@
     "logo": "https://symfony.com/images/v5/pictos/demoapp.svg?v=4",
     "success_url": "/",
     "scripts": {
-        "postdeploy": "php bin/console doctrine:schema:create && php bin/console doctrine:fixtures:load -n"
+        "postdeploy": "php bin/console doctrine:schema:create && php bin/console doctrine:fixtures:load -n",
+        "test-setup": "php bin/console doctrine:schema:create && php bin/console doctrine:fixtures:load -n",
+        "test": "phpunit"
```

Comment:
`$(composer config bin-dir)` is on `$PATH` on Heroku, so it uses the right one. I can change it though.

This actually begs the related question if it might be a good best practice for this project to, in `composer.json`, have a `test` entry in `scripts` that says `phpunit`? Then people can run `composer test`, similar to how they'd do with NPM ;)